Doc/library/glob.rst

```diff
@@ -23,8 +23,10 @@ according to the rules used by the Unix shell, although results are returned in
 arbitrary order.  No tilde expansion is done, but ``*``, ``?``, and character
 ranges expressed with ``[]`` will be correctly matched.  This is done by using
 the :func:`os.scandir` and :func:`fnmatch.fnmatch` functions in concert, and
-not by actually invoking a subshell.  Note that unlike :func:`fnmatch.fnmatch`,
-:mod:`glob` treats filenames beginning with a dot (``.``) as special cases.
+not by actually invoking a subshell.
+
+Note that :mod:`glob` treats filenames beginning with a dot (``.``) as special
```

Comment:
"treats ... as special cases" is still not clear: it says something is different, but not what the difference is.